renderer/src/components/FilAddressForm.tsx

```diff
@@ -88,16 +88,25 @@ const FilAddressForm: FC<FilAddressFormProps> = ({ destinationAddress = '', save
           <span className="text-xs px-4">Save</span>
         </button>
         <button className={`flex flex-row items-end h-[30px] cursor-pointer group absolute mb-1 ml-4 ease-[cubic-bezier(0.85,0,0.15,1)] duration-500
-        ${destinationAddress.length > 30 ? 'left-[460px]' : 'left-[70px]'} ${(!internalEditMode && !transferMode) ? 'visible' : 'opacity-0'}`}
-          tabIndex={0} onClick={() => enableEditMode() }
+        ${destinationAddress.length > 5 ? 'left-[168px]' : 'left-[80px]'} ${(!internalEditMode && !transferMode) ? 'visible' : 'opacity-0'}`}
+          tabIndex={0} onClick={() => enableEditMode()}
           disabled={transferMode}>
           <EditIcon className="btn-icon-primary mr-1" />
           <span className='text-white hidden group-hover:block opacity-80 not-italic text-body-m font-body leading-[1.5rem]'>Edit</span>
         </button>
       </>
     )
   }
-
+  const renderDisplayAddress = () => {
+    if (!internalEditMode) {
+      if (inputAddr.length > 5) {
+        return `${inputAddr.slice(0, 6)} . . . ${inputAddr.slice(inputAddr.length - 6, inputAddr.length)}`
```

Comment:
@bajtos proposed using `&hellip;` for the 3 points, what do you think about that? It's shorter and semantically correct

Reply:
For this particular case, there's a space between each dot. I can replace it with 3x `&period;`, the full stop character, if you prefer the HTML code

Reply:
It's not about the HTML code. If it's a design decision to have spaces between the dots, let's keep it as is